akka-docs/rst/scala/http/introduction.rst

```diff
@@ -10,6 +10,18 @@ You get to pick the API level of abstraction that is most suitable for your appl
 This means that, if you have trouble achieving something using a high-level API, there's a good chance that you can get
 it done with a low-level API, which offers more flexibility but might require you to write more application code.
 
+Phylosophy
```

Comment:
Argh I remembered the spelling wrong, so the PR subject was also wrong :| Fixed that in the following commit